pages/dkp/konvoy/2.2/release-notes/2.2.1/index.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: layout.pug
+navigationTitle: DKP 2.2.1 Release Notes
+title: DKP 2.2.1 Release Notes
+menuWeight: 20
+excerpt: View release-specific information for DKP 2.1.1
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Konvoy&reg; version 2.2.1 was released on May 26, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+Welcome to D2iQ Kubernetes Platform (DKP) 2.2.x! This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy. In this release, we are beginning the process of combining our two flagship products, Konvoy and Kommander, into a single DKP product with two service level options: DKP Enterprise for multi-cluster environments, and DKP Essential for single-cluster environments.
+
+For this release, we are maintaining the documentation sets for individual platform components Konvoy and Kommander, while publishing some combined DKP documentation for processes, such as Upgrading DKP version.
+
+DKP 2.2.x supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using DKP 2.2.x must be running a Kubernetes version in this range.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.21.0  |
+| **Maximum**  | 1.22.x  |
+| **Default**  | 1.22.0  |
```

Comment:
```suggestion
| **Default**  | 1.22.8  |
```

I believe the default version is 1.22.8

Reply:
See code: https://github.com/mesosphere/konvoy2/blob/v2.2.1-rc.3/pkg/api/v1alpha1/types.go#L15